psydac/fem/vector.py

```diff
@@ -163,7 +163,7 @@ def integral( self, f ):
         raise NotImplementedError( "VectorFemSpace not yet operational" )
 
     # ...
-    def pushforward_grid(self, grid, *fields, mapping=None, npts_per_cell=None):
+    def pushforward_fields(self, grid, *fields, mapping=None, npts_per_cell=None):
```

Comment:
If you want `mapping` to be a required keyword-only argument, you can do so by adding the `*` separator before it, and removing the default value `None`:
```
def pushforward_fields(self, grid, *fields, *, mapping, npts_per_cell=None):
```

Reply:
I checked and I can't do it because I have a starred argument `*fields` beforehand.

Reply:
I checked again and I can do it without adding the `*` because of the starred argument.

Reply:
Done in [b214861](https://github.com/pyccel/psydac/pull/212/commits/b214861020dd5759840cfd012919c8452089a99d)